osquery/utils/system/windows/env.cpp

```diff
@@ -54,4 +59,51 @@ boost::optional<std::string> getEnvVar(const std::string& name) {
   return std::string(buf.data(), value_len);
 }
 
+boost::optional<std::string> expandEnvString(const std::string& input) {
+  std::vector<char> buf;
+  buf.assign(kInitialBufferSize, '\0');
+
+  // ExpandEnvironmentStrings doesn't support inputs larger than 32k.
+  if (input.size() > 32768) {
+    return boost::none;
+  }
+
+  auto len =
+      ::ExpandEnvironmentStrings(input.c_str(), buf.data(), kInitialBufferSize);
+  if (len == 0) {
+    return boost::none;
```

Comment:
Same here. In the event `ExpandEnvironmentStrings` fails perhaps we should output the contents of `GetLastError`?